### PR TITLE
Add sidebar content and logic

### PR DESCRIPTION
**Pull request by andysellick**
Not well structured, but it seems to work.

- both task lists now built from partials, so we only have to change content in one place
- hard coded lists of URLs help to determine where we are and what steps to show
- highlighting now applied to sidebar
- session used to store which task list last looked at, in the event that a page occurs on both task lists (won't work in production), test using 'make arrangements for your children' in step 2 of both lists

Doesn't yet include highlighting for individual, current links.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -14,6 +14,115 @@ module ApplicationHelper
     ]
   }
 
+  CIVIL_URLS = [
```
Comment (Davidslv) [inline app/helpers/application_helper.rb]: I would move all these code from `ApplicationHelper` and would create a new one, maybe called `CivilPartnershipHelper` - nothing should change implementation wise as helpers are a _global_ thing in rails

File: app/views/content_items/show.html.erb [hunk 2]
```diff
@@ -22,16 +22,23 @@
...
-        </div>
-      <% else %>
-        <%= render partial: 'govuk_component/taxonomy_sidebar', locals: taxonomy_sidebar %>
+      <% if get_tasklist == 'divorce' %>
+        <%= render 'partials/get-a-divorce-sidebar' %>
+      <% elsif get_tasklist == 'civil' %>
+        <%= render 'partials/end-a-civil-partnership-sidebar' %>
+      <% end %>
+
+      <% if false %>
```
Comment (Davidslv) [inline app/views/content_items/show.html.erb]: what's the logic here?

Comment (andysellick) [reply app/views/content_items/show.html.erb]: Forgot about that - it's the simplest way I've found of commenting out chunks of code in rails. This bit can be deleted.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (andysellick) [reply app/helpers/application_helper.rb]: One for divorce and one for civil partnerships? Or just create a tasklistHelper?

File: app/views/content_items/show.html.erb (hunk 2 above)
Comment (andysellick) [reply app/views/content_items/show.html.erb]: Hey wait, just thought - is this why the sidebar isn't working on learn to drive anymore? Not the if false I added, but one of the lines within it was removed in a prior commit. Would probably impact it.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (Davidslv) [reply app/helpers/application_helper.rb]: One for both

File: app/views/content_items/show.html.erb (hunk 2 above)
Comment (Davidslv) [reply app/views/content_items/show.html.erb]: Maybe, I haven't got the chance to explore it yet. I'll have a look when I get a chance

**Pull request merged**